data/_ui/help.txt

```diff
@@ -141,7 +141,7 @@ help "navigation"
 
 help "outfitter"
 	`Here, you can buy new equipment for your ship. Your ship has a limited amount of "outfit space," and most outfits use up some of that space.`
-	`Some types of outfits have other requirements as well. For example, only some of your outfit space can be used for engines or weapons; this is your ship's "engine capacity" and "weapon capacity." Guns and missile launchers also require a free "gun port," and turrets require a free "turret mount." Also, missiles can only be bought if you have the right launcher installed.`
+	`Some types of outfits have other requirements as well. For example, only some of your outfit space can be used for engines or weapons; this is your ship's "engine capacity" and "weapon capacity." Guns and missile launchers also require a free "gun port," and turrets require a free "turret mount." Also, missiles can only be bought if you have the right launcher installed. Afterburners and other thruster modifications require a thruster to be attached to.`
```

Comment:
@endless-sky/writing-patrol 

This line is the only user-visible text change. I'm not set on the wording used here, I just feel that there should be an addition regarding it.

Alternative:
"Afterburners and other thruster modifications must be installed after the thruster they are attached to."

Reply:
One option would be to write some kind of "afterburners require an installed thruster" boilerplate text and stick it on all afterburners, like the "this ship requires a bay and lacks a hyperdrive etc" boilerplate on fighters and drones or the "this is ammunition, you can't install it unless you have something to fire it from" boilerplate on missiles.

Something along the lines of
`Afterburners cannot be installed on their own. They require a thruster to be installed first.`, but possibly slightly more eloquent.

Reply:
Yes, as per Hurleveur's comment, already planning on adding such a line. Do you think that would be sufficient?

Reply:
I think that would be sufficient to communicate the idea to players, yes. If it isn't, then we aren't communicating missile capacity or the concept of a fighter/drone well enough to the players currently either.

Reply:
> Yes, as per Hurleveur's comment, already planning on adding such a line. Do you think that would be sufficient?

As a player I think it would be sufficient.

Reply:
Agreeing that the boiler plate is best way to go, but that doesn't mean we shouldn't also include a reference in the tutorial teaching about it.

Reply:
Alright. I have added a boilerplate to all the afterburners except for the asteroid, which received an addendum noting that unlike regular afterburners, it *does* have the capacity to be installed by itself, as described in its description.